api/v1alpha1/storageconsumer_types.go

```diff
@@ -42,7 +42,18 @@ type StorageConsumerSpec struct {
 	Enable bool `json:"enable,omitempty"`
 	// StorageQuotaInGiB describes quota for the consumer
 	// +optional
-	StorageQuotaInGiB int `json:"storageQuotaInGiB,omitempty"`
+	StorageQuotaInGiB          int         `json:"storageQuotaInGiB,omitempty"`
+	RadosNamespace             string      `json:"radosNamespace,omitempty"`
+	SubVolumeGroup             string      `json:"subVolumeGroup,omitempty"`
```

Comment:
Should we call this `SubVolumeGroup` or `CephFsSubVolumeGroup`?

Reply:
I picked these names to be consistent with clientprofile fields https://github.com/ceph/ceph-csi-operator/blob/main/api/v1alpha1/clientprofile_types.go